cmd/ltcoordinator/main.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 func RunCoordinatorCmdF(cmd *cobra.Command, args []string) error {
-	cfg, err := coordinator.GetConfig()
+	configFilePath, _ := cmd.Flags().GetString("config")
+	cfg, err := coordinator.ReadConfig(configFilePath)
```

Comment:
Can we add a `strings.TrimSpace(configFilePath)` here, It results an error if we give a non existent program argument. E.g. `ltcoordinator -l $MM_COORDINATOR_CONFIG` where `$MM_COORDINATOR_CONFIG` is not set.